ctwrap/simulation.py

```diff
@@ -512,12 +512,12 @@ def run_task(self, sim: Simulation, task: str, **kwargs: str):
         :py:func:`~ctwrap.parser._save_metadata` to save metadata.
         A simple usage example is:
 
-        If a simulationHandler object ``sh`` was created from ``minimal.yaml``.
+        If a simulationHandler object ``sh`` was created from ``test_module.yaml``.
         This :meth:`run_task` method takes ``sim``, a simulation object created using
-        simulation module ``minimal``, and a task see :meth:`tasks` method.
+        simulation module ``test_module``, and a task see :meth:`tasks` method.
         This :meth:`run_task` method runs the task calling the ``run`` method in
-        simulation module ``minimal``, save the resulting output using
-        the ``save``  method in simulation module ``minimal`` and finally
+        simulation module ``test_module``, save the resulting output using
+        the ``save``  method in simulation module ``test_module`` and finally
         save the `metatada` using :py:func:`~ctwrap.parser.save_metadata` method.
```

Comment:
I believe some of this explanation is better moved to the docstring of the class itself - this avoids redundant descriptions here and in `run_serial` and `run_parallel`